recipes/jb-everything/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "jb-everything" %}
+{% set version = "0.1.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/jb-everything-{{ version }}.tar.gz
+  sha256: efe53c547aae8794372817182db86822e039349ad6099ce4c9f0cd2e37f1c21a
+
+build:
+  number: 0
+  skip: True  # [win]
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - dabl ~=0.1.9
+    - dask[complete] ~=2.30.0  # to be upstreamed
+    - eli5 ~=0.10.1
+    - jax ~=0.2.7
+    - lightgbm ~=3.1.0
+    - matplotlib-base ~=3.3.3
+    - missingno ~=0.4.2
+    - numpy ~=1.19.4
+    - pandas ~=1.1.4
+    - python >=3.5  # to be upstreamed
+    - requests ~=2.25.0
+    - scikit-learn ~=0.23.2
+    - scipy ~=1.5.3
+    - seaborn ~=0.11.0
+    - xarray ~=0.16.2
+
+test:
+  imports:
+    - jb_everything
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://pypi.org/project/jb-everything/
+  summary: Large list of imports for machine learning.
+  license: MIT
+  license_file: LICENSE  # to be upstreamed
```

Comment:
I opened a PR (https://github.com/jeromebaum/jb-everything/pull/1) to fix the need to manually include the license file here. Once it is in the source distribution, be sure to remove the local copy. I often recommend putting a comment in the `meta.yaml` as a reminder along the lines of:

```yaml
# License file manually packaged. See https://github.com/jeromebaum/jb-everything/pull/1
license_file: LICENSE
```

as a visual reminder for future updates. Also note that upstream the file is `LICENSE.txt` not `LICENSE`